src/event/entity/EntityExhaustEvent.php

```diff
@@ -47,20 +46,13 @@ class PlayerExhaustEvent extends EntityEvent implements Cancellable{
 	public const CAUSE_CUSTOM = 11;
 
 	public function __construct(
-		protected Human $human,
+		Human $human,
```

Comment:
Since the event name implies this is no longer limited to Humans, it would be nice to remove this limitation from the __construct so that other entities can use the event.